### PR TITLE
feat: add toString method and network information

### PR DESCRIPTION
**Pull request by alanshaw**
`Address` `network` added as an optional parameter.

closes #30

### CODE REVIEW

**Review by Schwartz10 — CHANGES_REQUESTED** [review]
Hey thanks for the quick turnaround! I like this approach a lot better.

Some tiny nits just to make sure we don't accidentally confuse anyone with ts semantics and/or anyone using the `Address` constructor directly. Also, how about I make you a maintainer of this repo, so I can more efficiently make PRs directly into your PRs for tiny nits like these? Let me know if you'd like that.

1. Let's mark `network` in the [constructor](https://github.com/glifio/modules/pull/37/files#diff-9e078ce1fdd0b69a572c12aa61efe6391641c8c651742480cc1416fbd46a3f97R19) as optional (I'm kind of surprised ts compiler didn't complain about setting a default value for a non-optional param! Something smelly... not for you to worry about). The built js files _clearly_ wouldn't introduce a breaking change here, but probably good for semantics anyways.
2. Another semantics-only request, let's add the default value for the [optional `network` param in `newAddress`](https://github.com/glifio/modules/pull/37/files#diff-9e078ce1fdd0b69a572c12aa61efe6391641c8c651742480cc1416fbd46a3f97R69) too. Future maintainers of this repo might thank us in case it makes it a bit more readable. 

As soon as I merge this, I'll publish it as `1.1.0-beta.16`. Hoping to remove the beta tags pretty soon.

**Comment by alanshaw** [comment]
Screenshot 2021-01-13 at 18 32 30

I think by virtue of having a default value it _is_ optional.

**Review by Schwartz10 — APPROVED** [review]
Aha! That's the ts error I was confused about. Merging and releasing this now

**Pull request merged**
